src/main/java/seedu/address/model/person/Person.java

```diff
@@ -19,11 +19,15 @@
     private final Phone phone;
     private final Email email;
     private final Address address;
+    //private final Location location;
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.